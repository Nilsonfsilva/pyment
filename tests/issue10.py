# -*- coding: utf-8 -*-

__version__ = '1.3.1'
# $Source$

#Some commentary


from .mod1 import *
#import mod2
from . import mod3, mod4


def init_test():
    '''
    Only working when in WinPython root directory.

    Returns:
    maske, data0, data1, no_motion, motion
    '''
    maske = read_mask('./MRR/Testdaten/55_mask.bmp')
    display(maske)
    data0 = read_dicom_set('./MRR/Testdaten/188_13-12-10_56_1',
                               unwrap=True, mask=maske, verbose=False)
    data1 = read_dicom_set('./MRR/Testdaten/188_13-12-10_54_1',
                               unwrap=True, mask=maske, verbose=False)
    no_motion = mean(data0, axis=0)
    motion = mean(data1, axis=0)
    return maske , data0, data1, no_motion, motion