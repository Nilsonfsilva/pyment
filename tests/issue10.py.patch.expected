--- a/issue10.py
+++ b/issue10.py
@@ -12,12 +12,12 @@
 
 
 def init_test():
-    '''
-    Only working when in WinPython root directory.
+    """Only working when in WinPython root directory.
 
-    Returns:
-    maske, data0, data1, no_motion, motion
-    '''
+
+    :returns: maske, data0, data1, no_motion, motion
+
+    """
     maske = read_mask('./MRR/Testdaten/55_mask.bmp')
     display(maske)
     data0 = read_dicom_set('./MRR/Testdaten/188_13-12-10_56_1',
